examples/gallery/images/grdclip.py

```diff
@@ -0,0 +1,35 @@
+"""
+Clip grid values
+----------------
+The :meth:`pygmt.grdclip` method allows to clip defined ranges of grid values.
+In the example shown below we set all elevation values (grid points) smaller
+than 0 m (in general the bathymetric part of the grid) to a common value of
+-2000 m via the ``below`` parameter.
+"""
+
+import pygmt
+
+fig = pygmt.Figure()
+
+# Load sample grid (1 arc minute global relief) and use area around the Hawaiian Islands
+grid = pygmt.datasets.load_earth_relief(resolution="01m", region=[-162, -153, 18, 23])
+
+# Plot original grid
+fig.basemap(
+    region=[-162, -153, 18, 23], projection="M12c", frame=["f", '+t"original grid"']
+)
+fig.grdimage(grid=grid, cmap="oleron")
+
+fig.shift_origin(yshift="-9c")
```

Comment:
Why not use `subplot`?

Reply:
In my opinion the whole example looks less crowded when using ```fig.shift_origin()``` here. However, I'm open to adjust it.

Reply:
I think both are okay. But, do you think it's better to compare the two figures with a 1X2 grid layout (left-right) instead of the current 2X1 (top-bottom)?

Reply:
> do you think it's better to compare the two figures with a 1X2 grid layout (left-right) instead of the current 2X1 (top-bottom)?

I think a 1x2 layout is better. The current 2x1 layout makes the image too long to display without scrolling down the page.

Reply:
Made some adjustements, thanks for all the input 😉